ansible_base/authentication/authenticator_plugins/base.py

```diff
@@ -6,10 +6,10 @@
 from rest_framework.reverse import reverse
 from rest_framework.serializers import ValidationError
 
-from ansible_base.models import Authenticator
-from ansible_base.serializers.fields import JSONField
+from ansible_base.authentication.models import Authenticator
```

Comment:
:+1: this move was expected. Having `ansible_base.models` is exactly what was becoming untenable.